.github/workflows/guitest.yml

```diff
@@ -14,7 +14,7 @@ on:
         required: false
 
       enable-profiling:
-        default: true
+        default: false
```

Comment:
I suppose that the problem is solved, and that is why profiling for GUI tests has been disabled.